docs/easymod/em_mgn12.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Easymod mgn12
+---
+
+> Created by: [EOliveira](https://github.com/EduardoOliveira)
+
+An implementation of easymod 1.2.1 using mgn12
+
+The XY joiners are drop in replacement for the official ones.
+
+This will allow you to use your setup with mgn12 in the X axis, mainly redugin the weight of it.
+
+[Onshape](https://cad.onshape.com/documents/8559b0e7f5d3b30427c227c5/w/d70f098c5b327e06a1ae23f4/e/12a2436a47701eb37945bb45?configuration=List_ByYG67Ay22XcMs%3DMGN12)
+
+![xyjoinners](assets/xyjoinners.png)
+
+
+If you are also using mgn12C or MGN12H on the Y axis you'll need the following adapters in place of the original aluminum plates.
+
+You'll need longer M5 screws as this adapters are 3mm thicker than the aluminum part. I recomend printing with more tops/bottoms and higher infill to incease strength.
```

Comment:
change `as this adapters` to `as these adapters` and `I recomend` to `I recommend`.